tests/libcxx/expected_results.txt

```diff
@@ -915,9 +881,6 @@ std/containers/sequences/vector/vector.cons/deduct.pass.cpp FAIL
 std/iterators/iterator.primitives/iterator.operations/advance.pass.cpp SKIPPED
 
 # Not yet analyzed. Maybe Clang over-eagerly instantiating noexcept-specifier?
-std/ranges/range.access/empty.pass.cpp:1 FAIL
-std/ranges/range.access/size.pass.cpp:1 FAIL
-std/ranges/range.access/ssize.pass.cpp:1 FAIL
 std/utilities/memory/unique.ptr/iterator_concept_conformance.compile.pass.cpp:1 SKIPPED

```

Comment:
Could this test be unskipped?

Reply:
Good question - the answer is no, it still emits a warning-as-error.

<details>
<summary>Click to expand warning:</summary>

```
D:\GitHub\STL\out\build\x64>python %STL%\tools\scripts\print_failures.py testing_x64.log
code: FAIL
name: libc++ :: std/utilities/memory/unique.ptr/iterator_concept_conformance.compile.pass.cpp:1
output: Build setup steps:
Build steps:
Build step failed unexpectedly.
Command: "C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\Llvm\x64\bin\clang-cl.EXE" "-c" "D:\GitHub\STL\llvm-project\libcxx\test\std\utilities\memory\unique.ptr\iterator_concept_conformance.compile.pass.cpp" "-ID:\GitHub\STL\out\build\x64\out\inc" "-ID:\GitHub\STL\llvm-project\libcxx\test\support" "/nologo" "/Od" "/W4" "/w14061" "/w14242" "/w14265" "/w14582" "/w14583" "/w14587" "/w14588" "/w14749" "/w14841" "/w14842" "/w15038" "/w15214" "/w15215" "/w15216" "/w15217" "/sdl" "/WX" "/Zc:strictStrings" "/D_ENABLE_STL_INTERNAL_CHECK" "/bigobj" "/EHsc" "/MTd" "/std:c++latest" "/permissive-" "/FImsvc_stdlib_force_include.h" "/wd4643" "/D_STL_CALL_ABORT_INSTEAD_OF_INVALID_PARAMETER" "/D_SILENCE_ALL_CXX23_DEPRECATION_WARNINGS" "/D_SILENCE_NONFLOATING_COMPLEX_DEPRECATION_WARNING" "/D_USE_JOIN_VIEW_INPUT_RANGE" "-fno-ms-compatibility" "-fno-delayed-template-parsing" "-m64"
Exit Code: 1
Standard Error:
--
In file included from D:\GitHub\STL\llvm-project\libcxx\test\std\utilities\memory\unique.ptr\iterator_concept_conformance.compile.pass.cpp:14:
D:\GitHub\STL\out\build\x64\out\inc\memory(3264,93): error: ISO C++ does not allow indirection on operand of type 'void *' [-Werror,-Wvoid-ptr-dereference]
    _NODISCARD _CONSTEXPR23 add_lvalue_reference_t<_Ty> operator*() const noexcept(noexcept(*_STD declval<pointer>())) {
                                                                                            ^~~~~~~~~~~~~~~~~~~~~~~~
D:\GitHub\STL\out\build\x64\out\inc\__msvc_iter_core.hpp(215,11): note: in instantiation of exception specification for 'operator*' requested here
        { *__i } -> _Can_reference;
          ^
D:\GitHub\STL\out\build\x64\out\inc\__msvc_iter_core.hpp(215,11): note: in instantiation of requirement here
        { *__i } -> _Can_reference;
          ^~~~
D:\GitHub\STL\out\build\x64\out\inc\__msvc_iter_core.hpp(214,5): note: while substituting template arguments into constraint expression here
    requires(_It __i) {
    ^~~~~~~~~~~~~~~~~~~
D:\GitHub\STL\out\build\x64\out\inc\__msvc_iter_core.hpp(233,40): note: while checking the satisfaction of concept '_Cpp17_iterator<std::unique_ptr<void>>' requested here
    requires (!_Has_iter_types<_It> && _Cpp17_iterator<_It> && !_Cpp17_input_iterator<_It>)
                                       ^~~~~~~~~~~~~~~~~~~~
D:\GitHub\STL\out\build\x64\out\inc\__msvc_iter_core.hpp(233,40): note: while substituting template arguments into constraint expression here
    requires (!_Has_iter_types<_It> && _Cpp17_iterator<_It> && !_Cpp17_input_iterator<_It>)
                                       ^~~~~~~~~~~~~~~~~~~~
D:\GitHub\STL\out\build\x64\out\inc\__msvc_iter_core.hpp(377,26): note: (skipping 7 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
struct iterator_traits : _Iterator_traits_base<_Ty> {
                         ^
D:\GitHub\STL\out\build\x64\out\inc\xutility(442,5): note: in instantiation of requirement here
    typename iter_value_t<_It>;
    ^~~~~~~~~~~~~~~~~~~~~~~~~~
D:\GitHub\STL\out\build\x64\out\inc\xutility(441,37): note: while substituting template arguments into constraint expression here
concept _Indirectly_readable_impl = requires(const _It __i) {
                                    ^~~~~~~~~~~~~~~~~~~~~~~~~
D:\GitHub\STL\out\build\x64\out\inc\xutility(452,31): note: while checking the satisfaction of concept '_Indirectly_readable_impl<std::unique_ptr<void>>' requested here
concept indirectly_readable = _Indirectly_readable_impl<remove_cvref_t<_It>>;
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
D:\GitHub\STL\out\build\x64\out\inc\xutility(452,31): note: while substituting template arguments into constraint expression here
concept indirectly_readable = _Indirectly_readable_impl<remove_cvref_t<_It>>;
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
D:\GitHub\STL\llvm-project\libcxx\test\std\utilities\memory\unique.ptr\iterator_concept_conformance.compile.pass.cpp(25,16): note: while checking the satisfaction of concept 'indirectly_readable<std::unique_ptr<void>>' requested here
static_assert(!std::indirectly_readable<std::unique_ptr<void> >);
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
--
```
</details>

It's SKIPPED instead of FAIL due to #2621:

> * In `tests/libcxx/expected_results.txt`, change 2 tests from `FAIL` to `SKIPPED`. This avoids unexpected passes when running `set_environment` before running the tests. (Because `set_environment` puts the freshly-built STL on the `INCLUDE` path, Clang suppresses warnings from those "system headers", so if tests are expected to `FAIL` due to such warnings, they will instead unexpectedly pass.)
>   + This is a quality-of-life improvement for myself, as I always run `set_environment`.